src/test/java/se/sundsvall/billingdatacollector/model/BillingRecordWrapperTest.java

```diff
@@ -46,10 +48,11 @@ void testBuilderMethods() {
 		assertThat(wrapper.getFamilyId()).isEqualTo(familyId);
 		assertThat(wrapper.getFlowInstanceId()).isEqualTo(flowInstanceId);
 		assertThat(wrapper.getMunicipalityId()).isEqualTo(municipalityId);
+		assertThat(wrapper.isRecipientPrivatePerson()).isEqualTo(isRecipientPrivatePerson);
 	}
 
 	@Test
 	void testNoDirtOnCreatedBean() {
-		assertThat(BillingRecordWrapper.builder().build()).isNotNull().hasAllNullFieldsOrProperties();
+		assertThat(BillingRecordWrapper.builder().build()).isNotNull().hasAllNullFieldsOrPropertiesExcept("isRecipientPrivatePerson");
```

Comment:
add `.hasFieldOrPropertyWithValue("isRecipientPrivatePerson", false)` before `;` to verify that attribute is not tampered with on creation?